Sources/protoc-gen-swift/MessageFieldGenerator.swift

```diff
@@ -275,10 +275,12 @@ extension Google_Protobuf_FieldDescriptorProto {
 struct MessageFieldGenerator {
     private let fieldDescriptor: FieldDescriptor
     private let generatorOptions: GeneratorOptions
+    private let namer: SwiftProtobufNamer
 
     var descriptor: Google_Protobuf_FieldDescriptorProto { return fieldDescriptor.proto }
     let oneof: Google_Protobuf_OneofDescriptorProto?
     let jsonName: String?
+    let hasFieldPresence: Bool
```

Comment:
So I'm mixed about this.  Making it a computed property means the call go through every time it is used.  Making it a let and assigning it within init means it is called once and cached locally.  So there's actually a big difference in what happens under the hood.

I'd love it if there was a way that wasn't `lazy` to say "this value comes from this sub object", and not force the split world of the decl and collecting in init.